src/lavinmq/exchange/mqtt.cr

```diff
@@ -32,42 +32,46 @@ module LavinMQ
       super(vhost, name, true, false, true)
     end
 
+    def publish(msg : Message, immediate : Bool,
+                queues : Set(Queue) = Set(Queue).new,
+                exchanges : Set(Exchange) = Set(Exchange).new) : Int32
+      raise LavinMQ::Exchange::AccessRefused.new(self)
```

Comment:
@snichme Now `spec/message_routing_spec.cr:439` is failing 
`Access refused to mqtt.default (LavinMQ::Exchange::AccessRefused)`

Reply:
I think it should fail, I'll rewrite the specs